kubevirt_handler/metadata.csv

```diff
@@ -0,0 +1,47 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
+kubevirt_handler.can_connect,gauge,,,,Whether the check can connect to the KubeVirt Handler or not.,0,kubevirt_handler,handler connect,,
+kubevirt_handler.info,gauge,,,,"KubeVirt Version information",0,kubevirt_handler,short_name,,
```

Comment:
Would this better be served as a metadata entry instead of a metric?

https://github.com/DataDog/integrations-core/blob/master/argo_rollouts/datadog_checks/argo_rollouts/metrics.py#L70

Reply:
For some reason, It's not working for me, do we have any documentation about it?
The kubevirt info metric looks like:
```
# HELP kubevirt_info Version information
# TYPE kubevirt_info gauge
kubevirt_info{goversion="go1.21.8 X:nocoverageredesign",kubeversion="v1.2.2"} 1

```

Reply:
It's fixed, I had to use a metrics transformer.